packages/app/src/messages/selectors/get-thread-draft-messages.selector.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { createSelector } from "reselect"
+import { MessageType } from "App/messages/constants"
+import { Message } from "App/messages/dto"
+import { messagesStateSelector } from "App/messages/selectors/messages-state.selector"
+
+export const getThreadDraftMessagesSelector = (threadId: string) =>
```

Comment:
It returns one message, but the name indicates that it returns multiple.
Shouldn't it be then `getThreadDraftMessageSelector`?

Reply:
https://github.com/mudita/mudita-center/pull/835/commits/31b4aae7dbdc63e21c41fb1e179a2ea58b9d9d98